src/perception/tag_detector/long_range_tag_detector.cpp

```diff
@@ -0,0 +1,156 @@
+#include "tag_detector.hpp"
+
+namespace mrover {
```

Comment:
you may want to make a commit to remove `src/perception/tag_detector` from this branch